src/directsketch.rs

```diff
@@ -115,6 +115,15 @@ async fn download_and_parse_md5(client: &Client, url: &Url) -> Result<HashMap<St
         .await
         .context("Failed to send request")?;
 
+    // Check if the status code is 200 OK
+    if !response.status().is_success() {
```

Comment:
idle curiousity: does `reqwest` allow you to automatically follow HTTP redirects? Is that enabled?

Reply:
I think it's automatic
> By default, a Client will automatically handle HTTP redirects, having a maximum redirect chain of 10 hops. To customize this behavior, a redirect::Policy can be used with a ClientBuilder.
https://docs.rs/reqwest/latest/reqwest/redirect/index.html#

Reply:
fantastic!